tools/v8_gypfiles/v8.gyp

```diff
@@ -2536,16 +2536,24 @@
           },
         }],
         ['OS=="aix"', {
+          'variables': {
+            # Used to differentiate `AIX` and `OS400`(IBM i).
+            'aix_variant_name': '<!(uname -s)',
+          },
           'sources': [
             '<(V8_ROOT)/src/base/debug/stack_trace_posix.cc',
             '<(V8_ROOT)/src/base/platform/platform-aix.cc',
           ],
-          'link_settings': {
-            'libraries': [
-              '-ldl',
-              '-lrt'
-            ],
-          },
+          'conditions': [
+            [ '"<(aix_variant_name)"!="OS400"', { # It is `AIX`
```

Comment:
I would prefer it if you write this as `'"<(aix_variant_name)"=="AIX"'`, that's the style we use in common.gypi.

Reply:
Good idea!